scripts/deps-ppa/static_z3.sh

```diff
@@ -226,26 +240,26 @@ cd ..
 orig="${packagename}_${debversion}.orig.tar.gz"
 # shellcheck disable=SC2012
 orig_size=$(ls -l "$orig" | cut -d ' ' -f 5)
-orig_sha1=$(sha1sum $orig | cut -d ' ' -f 1)
-orig_sha256=$(sha256sum $orig | cut -d ' ' -f 1)
-orig_md5=$(md5sum $orig | cut -d ' ' -f 1)
+orig_sha1=$(sha1sum "$orig" | cut -d ' ' -f 1)
+orig_sha256=$(sha256sum "$orig" | cut -d ' ' -f 1)
+orig_md5=$(md5sum "$orig" | cut -d ' ' -f 1)
 
-if wget --quiet -O $orig-tmp "$ppafilesurl/$orig"
+if wget --quiet -O "$orig-tmp" "$ppafilesurl/$orig"
 then
     echo "[WARN] Original tarball found in Ubuntu archive, using it instead"
-    mv $orig-tmp $orig
+    mv "$orig-tmp" "$orig"
```

Comment:
If you're adding quotes I'd also add braces. It looks a bit as if `orig-tmp` was the variable name.
```suggestion
    mv "${orig}-tmp" "$orig"
```